src/main/java/co/inventorsoft/academy/collections/model/Range.java

```diff
@@ -1,60 +1,83 @@
 package co.inventorsoft.academy.collections.model;
 
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.Set;
+import java.util.*;
 
 public class Range<T> implements Set<T> {
 
+    private static List  mainList = new ArrayList();
+
     public int size() {
-        return 0;
+        return mainList.size();
     }
 
     public boolean isEmpty() {
-        return false;
+        return mainList.isEmpty();
     }
 
     public boolean contains(Object o) {
-        return false;
+        return mainList.contains(o);
     }
 
     public Iterator<T> iterator() {
-        return null;
+        return mainList.iterator();
     }
 
     public Object[] toArray() {
-        return new Object[0];
+        return mainList.toArray();
     }
 
     public <T1> T1[] toArray(T1[] a) {
         return null;
     }
 
     public boolean add(T t) {
-        return false;
+        return mainList.add(t);
```

Comment:
Still I believe that would allow duplications

Reply:
Fixed